core/core_modules/issue.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Creates an issue on github for kassy.
+ *
+ * Written By:
+ * 		Dion Woolley
+ *
+ * License:
+ *		MIT License. All code unless otherwise specified is
+ *		Copyright (c) Dion Woolley and Contributors 2016.
+ */
+
+var command = "issue",
+    githubAPI = "https://api.github.com/repos/mrkno/Kassy/issues",
+    token = 'ZWZkN2ZiMGY2OWViZDlkNmQyZmE0Yzc4ODFjNTkyY2JmNmQxYTljNA==',
+    request = require.safe('request'),
+    fs = require('fs'),
+    config = null;
+
+try {
+    config = require.once('../../config.json');
+}
+catch (e) {
+    //If config file does not exist.
+}
+
+var submitRequest = function(title, description, debugLevel, callback, waitCallback) {
+
+        waitCallback();
+
+        var data = {title: title},
+            debugData = null;
+
+        if (description !== null) {
+            data.body = description;
+        }
+        else {
+            data.body = '';
+        }
+        if (debugLevel === 'full') {
+            var cache = [];
+            data.body += "\nfull debug details: " + JSON.stringify(this, function (key, value) {
+                    if (typeof value === 'object' && value !== null) {
+                        if (cache.indexOf(value) !== -1) {
+                            // Circular reference found, discard key
+                            return;
+                        }
+                        // Store value in our collection
+                        cache.push(value);
+                    }
+                    return value;
+                }, 2);
+            cache = null;
+        }
+        if (debugLevel === 'detail' || debugLevel === 'full') {
+            var stats = null;
+            if (config && config !== null) {
+                data.body += "\nconfig: ";
+                for (var key in config) {
+                    if (key !== 'output') {
+                        data.body += key + ":" + JSON.stringify(config[key], null, 4);
+                    }
+                }
+            }
+            try{
+                stats = fs.statSync(filePath);
+            }catch(err){
+                //if file does not exist.
```

Comment:
fair enough, I think i'll change it to just try and read the file outright instead of bothering to check the file exists